backend/app/api/endpoints/openapi_responses.py

```
@@ -145,7 +148,1095 @@ def _task_to_response_object(
+                await db_handler.update_subtask_status(
+                    assistant_subtask_id, "FAILED", error=str(e)
+                )
+            except Exception:
+                pass
+            raise
+        finally:
+            # Cleanup MCP client if used
+            if mcp_client:
+                try:
+                    await mcp_client.close()
+                except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)